aptos-move/framework/supra-framework/sources/supra_governance.move

```diff
@@ -217,6 +290,29 @@ module supra_framework::supra_governance {
         })
     }
 
+    fun supra_initialize(
```

Comment:
Use `initialize` name only so that vm erroneously doesn't call the incorrect version. Remove the other/older version of the function.